app/policies/business_category_policy.rb

```diff
@@ -4,30 +4,24 @@ def new?
     user.admin?
   end
 
-
```

Comment:
We don't have a specific coding style standard for the project.   
There is a bit of variation throughout the code.  We generally accept some different formatting choices / coding styles as long as it's readable.

But in general, we do accept 2 (or more) blank lines between methods, etc.  [That's actually my preference.]

In general, unless someone has said they have a specific issue or strong preference with some coding formatting or style, we don't change it.  If you do think it should be changed, then put a note about your reasoning in the PR description or your code, or just bring it up in the Slack #shf-project channel.

Since we don't have any coding standards written, you had no way to know about any of this of course.  Do you think we should write up a little something?

So - undo these changes, please. return them to 2 blank lines between methods.